string mutation
string[] manufacturer
string[] typeNumber
string[] serialNumber
